pkg/services/grafana-apiserver/service.go

```diff
@@ -316,20 +314,17 @@ func (s *service) start(ctx context.Context) error {
 		}
 	}
 
+	// Used by the proxy wrapper registered in ProvideService
+	s.handler = server.Handler
 	s.restConfig = server.LoopbackClientConfig
 
 	// only write kubeconfig in dev mode
 	if s.config.devMode {
 		if err := s.ensureKubeConfig(); err != nil {
 			return err
 		}
-	}
-
-	// Used by the proxy wrapper registered in ProvideService
-	s.handler = server.Handler
-
-	// skip starting the server in prod mode
-	if !s.config.devMode {
+	} else {
```

Comment:
would it make sense to invert the if statement above and return nil on line 323?

Reply:
updated -- good point.  I think that is more clear